.github/workflows/create_release.yml

```diff
@@ -1,6 +1,8 @@
 name: Deploy to Wordpress and create new release
 on:
-  workflow_dispatch: {}
+  push:
+    tags:
+    - "v*"
```

Comment:
Please keep the manual launch of the workflow

Reply:
Done